openshift/nginx-configmap.yml

```diff
@@ -0,0 +1,148 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: childcarerecalc-config
+  namespace: licenseplate-env
+data:
+  nginx.conf: |
+    error_log  /var/log/nginx/error.log;
+    events {
+      worker_connections  4096;
+    }
+    http {
+      log_format  main
+        'remote_addr:$remote_addr\t'
+        'time_local:$time_local\t'
+        'method:$request_method\t'
+        'uri:$request_uri\t'
+        'host:$host\t'
+        'status:$status\t'
+        'bytes_sent:$body_bytes_sent\t'
+        'referer:$http_referer\t'
+        'useragent:$http_user_agent\t'
+        'forwardedfor:$http_x_forwarded_for\t'
+        'request_time:$request_time';
+      access_log	/var/log/nginx/access.log main;
+      include /etc/nginx/conf.d/*.conf;
+      server {
+        listen      8080;
+        server_name  _ *.childcarerecalc.gov.bc.ca;
+
+        include /etc/nginx/conf.d/*.conf;
+
+        location / {
+
+          root   /usr/share/nginx/html;
+
+          index  index.html index.htm;
+
+          try_files $uri $uri/ /index.html;
+
+          include /etc/nginx/mime.types;
+
+
+          add_header Strict-Transport-Security "max-age=31536000; includeSubdomains;
+          preload" always;
+
+          add_header X-Content-Type-Options "nosniff" always;
+
+          add_header Referrer-Policy "no-referrer-when-downgrade";
+
+          add_header Access-Control-Allow-Credentials "true";
+
+          #add_header  X-Frame-Options "SAMEORIGIN" always;
+
+          add_header X-Frame-Options "DENY";
+          add_header Content-Security-Policy "default-src 'self' https://sfstest7.gov.bc.ca *.gov.bc.ca *.childsupportrecalc.gov.bc.ca *.devops.gov.bc.ca *.oidc.gov.bc.ca https://spt.apps.gov.bc.ca https://spm.apps.gov.bc.ca; frame-src 'self' *.gov.bc.ca;  script-src 'self' *.childsupportrecalc.gov.bc.ca https://www2.gov.bc.ca  *.gov.bc.ca ; style-src 'self' ; font-src 'self' *.gov.bc.ca https://fonts.googleapis.com https://fonts.gstatic.com; img-src 'self'; connect-src 'self' *.gov.bc.ca; manifest-src 'self';" always;
```

Comment:
will get rid of *.gov.bc.ca when I have prod equivalent of sfstest7 (as for example) handy